extensions/mssql/src/objectExplorerNodeProvider/hdfsProvider.ts

```diff
@@ -86,7 +86,7 @@ export abstract class HdfsFileSourceNode extends TreeNode {
 		return this._path;
 	}
 
-	public get nodePathValue(): string {
+	public get nodePath(): string {
```

Comment:
Please undo this change. The issue here is that sqlops.NodeInfo has a `nodePath` that is of the form `parent/child/subchild/subsubchild`. This `nodePathValue` is 1 of the sections in this overall `nodePath`. e.g. it'll be `parent` or `child`, or some other value in the overall path.